tests/tests.rs

```diff
@@ -272,6 +272,34 @@ fn dynamic_components() {
     );
 }
 
+#[test]
+fn spawn_buffered_entity() {
+    let mut world = World::new();
+    let mut buffer = CommandBuffer::new();
+    let ent = world.reserve_entity();
+    let ent1 = world.reserve_entity();
+    let ent2 = world.reserve_entity();
+    let ent3 = world.reserve_entity();
+
+    buffer.spawn_at(ent, (1, true));
+    buffer.spawn_at(ent1, (7.11, 13 as usize, "hecs"));
+    buffer.spawn_at(ent2, (17 as i8, false, 'o'));
+    buffer.spawn_at(ent3, (2 as u8, "qwe", 101.103, false));
+
+    buffer.run_on(&mut world);
+
+    let be_true = world.get::<bool>(ent).unwrap();
+    let be_hecs = world.get::<&str>(ent1).unwrap();
+    let be_false = world.get::<bool>(ent2).unwrap();
+    let be_two = world.get::<u8>(ent3).unwrap();
+
+    assert!(world.contains(ent));
```

Comment:
This `assert` is now redundant to the `unwrap()` above. As a style nit, I'd also prefer for the `get` calls to be inside the `assert_eq` invocations; this makes changes easier and less error-prone.